TotalCrossSDK/build.gradle

```diff
@@ -200,9 +170,10 @@ publishing {
 task sourcesJar(type: Jar, dependsOn: classes) {
 	baseName = "totalcross-sdk"
     classifier = 'sources'
-    from (sourceSets.desktop.allSource) { 
+    from (sourceSets.main.allSource) { 
     	exclude 'tc/**', 'ras/**', '**/*4A.java', '**/*4D.java'
```

Comment:
this is not something really serious, but we dont have any ras.* packages anymore